db/migrate/20150520144034_add_coupon_to_member.rb

```diff
@@ -0,0 +1,5 @@
+class AddCouponToMember < ActiveRecord::Migration
+  def change
+    add_column :members, :coupon, :string
```

Comment:
Looking at it, most of the changes are spacing. No idea why the `data_migrations` table has been dinged though. I thought that table was a Rails default anyway